src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/preferences.html.twig

```diff
@@ -165,6 +165,9 @@
                                     {{ form_widget(generalForm.shop_activity) }}
                                 </div>
                             </div>
+                            {% block shop_preferences_form_rest %}
+                              {{ form_rest(form) }}
```

Comment:
mixin 2 spaces & 4 spaces.
We need to do something for our templates :)

Reply:
Agree, but do you think it can be done in separate PR for all template files with some automatic tool?

Reply:
Yes of cour,se it's just a reminder :)

Reply:
https://github.com/PrestaShop/PrestaShop/issues/14146